source/gui/settingsDialogs.py

```diff
@@ -2858,6 +2866,163 @@ def onBlinkCursorChange(self, evt):
 	def onNoMessageTimeoutChange(self, evt):
 		self.messageTimeoutEdit.Enable(not evt.IsChecked())
 
+
+class VisionSettingsPanel(SettingsPanel):
+	# Translators: This is the label for the vision panel
+	title = _("Vision")
+
+	def makeSettings(self, settingsSizer):
+		self.initialProviders = list(vision.handler.providers)
+		self.providerPanelInstances = []
+		settingsSizerHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+
+		# Translators: The label for a setting in vision settings to choose a vision enhancement provider.
+		providerLabelText = _("&Vision enhancement providers")
+
+		# Translators: Description in vision settings to choose a vision enhancement provider.
+		providersListDescriptionText = _(
+			"Providers are activated and deactivated as soon as you check or uncheck check boxes."
+		)
+		providersSizer = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
+		providerListDescTextCtrl = providersSizer.addItem(wx.StaticText(self, label=providersListDescriptionText))
+		providerListDescTextCtrl.Wrap(self.scaleSize(544))
+		self.providerList = providersSizer.addLabeledControl(
+			f"{providerLabelText}:",
+			nvdaControls.CustomCheckListBox,
+			choices=[]
+		)
+		self.providerList.Bind(wx.EVT_CHECKLISTBOX, self.onProviderToggled)
+
+		self.initializeProviderList()
+
+		settingsSizerHelper.addItem(providersSizer, flag=wx.EXPAND)
+		settingsSizerHelper.addItem(
+			wx.StaticLine(self, size=wx.Size(-1, self.scaleSize(2))),

```

Comment:
Also, should -1 and 2 be constants?

Reply:
On second thoughts, you can remove the size parameter.